tests/test_image/test_io.py

```diff
@@ -28,6 +28,7 @@ def setup_class(cls):
         cls.palette_img_path = osp.join(cls.data_dir, 'palette.gif')
         cls.exif_img_path = osp.join(cls.data_dir, 'color_exif.jpg')
         cls.img = cv2.imread(cls.img_path)
+        cls.tiff_path = osp.join(cls.data_dir, 'uint16-5channel.tif')
```

Comment:
The file is 1.15MB large. Is it possible to use a smaller image?

Reply:
ok, I’ll replace it with smaller one.

Reply:
It's now 208Kb.